openmls/src/framing/mod.rs

```diff
@@ -90,6 +92,12 @@ pub enum WireFormat {
     PublicMessage = 1,
     /// Encrypted message
     PrivateMessage = 2,
+    /// Welcome message
```

Comment:
Unrelated to this PR but the IANA values in the spec still use plaintext and cihpertext.

Reply:
Yes, I noticed that, too. That's why I filed https://github.com/mlswg/mls-protocol/pull/844.